### ISSUE TITLE
Breaking change in dependency Cheerio v1.0.0-rc.9

### ISSUE THREAD

**Issue by jordanmaguire** [sourcey/spectacle#223]
One of the dependencies of spectacle is Cheerio

The latest RC has a breaking change.

https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0-rc.9

I suppose you might like to decide whether  you lock the dependency down to an old version or lock it down to a newer version and change the syntax to fix the breaking change.

In the interim, I'm working around this by locking the dependency myself with:

```json
  "resolutions": {
    "spectacle-docs/cheerio": "1.0.0-rc.6"
  },
```

**Comment by tommyschnabel** [comment]
+1

**Comment by jordanmaguire** [comment]
Looks to be resolved by #224 

`yarn add 'spectacle-docs@https://github.com/sourcey/spectacle#3257da500b8a2483b6ad3356e50fe00b65bb6a6f'` until there's a release
